pkg/log/zap/flags.go

```diff
@@ -29,16 +29,15 @@ import (
 )
 
 var levelStrings = map[string]zapcore.Level{
-	"debug":  zap.DebugLevel,
-	"-1":     zap.DebugLevel,
-	"info":   zap.InfoLevel,
-	"0":      zap.InfoLevel,
-	"error":  zap.ErrorLevel,
-	"2":      zap.ErrorLevel,
-	"dpanic": zap.DPanicLevel,
-	"panic":  zap.PanicLevel,
-	"warn":   zap.WarnLevel,
-	"fatal":  zap.FatalLevel,
+	"debug": zap.DebugLevel,
+	"info":  zap.InfoLevel,
+	"error": zap.ErrorLevel,
```

Comment:
Should we not keep the others levels? (`warn`, `panic`, `fatal`)

Reply:
@camilamacedo86 Those are not valid `logr` levels, so they can't be used when `zapr` is being used to implement the `logr` interface for zap loggers.